editor.planx.uk/src/pages/FlowEditor/floweditor.scss

```diff
@@ -137,6 +137,7 @@ $fontMonospace: "Source Code Pro", monospace;
     border: $nodeBorderWidth dashed red;
   }
 
+  & > div > a,
```

Comment:
Another reminder that it would be good to update these styles and incorporate them into MUI components sometime soon!

Reply:
:heavy_plus_sign: